src/translations/nbTranslations.json

```diff
@@ -2076,7 +2076,7 @@
     "new_tasks": "Innmeldte oppgaver: {{count}}",
     "nvi": {
       "approve_nvi_candidate": "Godkjenn resultat",
-      "approve_nvi_candidate_description": "<0>Kontroll av forskningsresultat for NVI-rapportering.</0><0>Klikk på knappen \"{{buttonText}}\" for å rapportere forskningsresultatet.</0>",
+      "approve_nvi_candidate_description": "<p>Kontroll av forskningsresultat for NVI-rapportering.</p><p>Trykk på \"{{buttonText}}\" om publikasjonen kan rapporteres. Se <hyperlink>Rapporteringsinstruksen<launchIcon/></hyperlink> for mer informasjon.</p>",
```

Comment:
Støtter i18next også denne syntaksen med å gi tags de navnene man vil??? Har hata alle `<0>`-greiene siden dag en, og jobba litt med `react-intl` i jula, som har akkurat dette og var en drøm å bruke. Men trodde ikke det kunne brukes med i18next 😅 
Det er faktisk en stor game changer for min del! 🎉

Reply:
Faktisk vært tilgjengelig siden [v11.6.0](https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0) som kom i [juni 2020](https://www.npmjs.com/package/react-i18next?activeTab=versions) 🫨